sys-utils/irq-common.c

```diff
@@ -568,7 +575,7 @@ struct libscols_table *get_scols_table(struct irq_output *out,
 	}
 
 	for (i = 0; i < stat->nr_irq; i++)
-		add_scols_line(out, &result[i], table);
+		add_scols_line(out, &result[i], table, threshold);
 
```

Comment:
It would be better to use 
```C
for (i = 0; i < stat->nr_irq; i++) {
   if (result[I].total < threshold)
      continue;
   add_scols_line(out, &result[i], table);
}
```

Reply:
Yes, it is better :)